### PR TITLE
Move perf_kit to tests.utils

### PR DESCRIPTION
**Pull request by potiuk**
Perf_kit was a separate folder and it was a problem when we tried to
build it from Docker-embedded sources, because there was a hidden,
implicit dependency between tests (conftest) and perf.

Perf_kit is now moved to tests to be avaiilable in the CI image
also when we run tests without the sources mounted.
This is changing back in #10441 and we need to move perf_kit
for it to work.

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

**Review by turbaszek — APPROVED** [review]

File: tests/utils/perf/perf_kit/__init__.py [hunk 1]
```diff
@@ -32,33 +32,34 @@
 =======
```
Comment (mik-laj) [inline tests/utils/perf/perf_kit/__init__.py:32]: The previous section should be updated as it contains invalid paths.

Comment (potiuk) [reply tests/utils/perf/perf_kit/__init__.py:32]: Ah yeah. Fixing it now.

**Comment by turbaszek** [comment]
@potiuk why we put it in `utils` not `test_utils`?

**Comment by potiuk** [comment]
@turbaszek - because both dirs were there and my eye caught "utils" first I guess :)

 Both directories were there when I moved it, so I guess arbitrarily I chose the "utils" one. 

But feel free to merge those two. There is no difference from the build POV.

**Comment by potiuk** [comment]
> The point is that `tests.utils` corresponds to `airflow.utils` and probably should have no other stuff. I will open PR

I  see, not too obvious though :( . I was never suspecting this. But I am not sure how to make it more obvious,

**Comment by turbaszek** [comment]
> > The point is that `tests.utils` corresponds to `airflow.utils` and probably should have no other stuff. I will open PR
> 
> I see, not too obvious though :( . I was never suspecting this. But I am not sure how to make it more obvious,

I think we should pay more attention to this in future. At least I will do 😄 

Screenshot 2020-09-11 at 22 37 40

**Comment by potiuk** [comment]
:rofl:

**Pull request merged**